### ISSUE TITLE
sqrt paste from wolfram not rendering properly

### ISSUE THREAD

**Issue by ionsme** [DesModder/DesModder#336]
https://www.wolframalpha.com/input?i=einstein+curve+parametric+form

I copied this output as plain text, and pasted it in to desmos.

Desmos interpererted sqrt() as variables s, q, r,t rather than a function
This was true regardless of whether radical Notation was turned on or not.

**Comment by Heavenira** [comment]
@ionsme This is intentional behavior from Wolfram2Alpha. Your clipboard contains a `\n` (newline) character, which aborts parsing altogether.

I created this `\n` countermeasure to protect users from pasting invalid clipboards.

For your situation, you must paste Albert Einstein's `x(t)` and `y(t)` functions separately, eliminating any `\n` characters. This is how I've pasted the Wolfram graphs in the past. [Example of this graph working.](https://www.desmos.com/calculator/h9e5upp0tr)
